Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] 
+    private int _speed = 3; 
+    Rigidbody2D rb2d;
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+       
+        if (Input.GetKey(KeyCode.A))
+        {
+            transform.position = transform.position + new Vector3(-(_speed)*Time.deltaTime,0,0);
```

Comment:
This line is not formatted, the values inside `Vector3.x` are too clunky, see Microsoft docs or auto format tool on your IDE.

Reply:
What do you mean about clunky?, it's too basic to use the values of 0,0?

Reply:
This code `new Vector3(-(_speed)*Time.deltaTime,0,0)` does not follow the C# guidelines for operator separation, this should be like this `new Vector3(-_speed * Time.deltaTime, 0, 0)`, there are some rules that all the code in C# Microsoft docs respect, like space after commas on parameters or space on each side on math or boolean operators.

Reply:
ohhh okay, thank you